packages/plugin-ext/src/main/browser/workspace-main.ts

```diff
@@ -367,9 +367,7 @@ export class TextContentResource implements Resource {
             return content;
         } else {
             const content = await this.proxy.$provideTextDocumentContent(this.uri.toString());
-            if (content) {
-                return content;
-            }
+            return content ?? '';
         }
 
         return Promise.reject(new Error(`Unable to get content for '${this.uri.toString()}'`));
```

Comment:
Minor: This `return` statement is unreachable, so we should remove it.

Reply:
@msujew @vince-fugnitto Thank you for your review. I have resolve it.